node-backend/google_auth/google_auth.js

```diff
@@ -0,0 +1,82 @@
+"use strict";
+var passport = require("passport");
+const pool = require("../database/database");
+var GoogleTokenStrategy = require("passport-google-token").Strategy;
+var config = require("../config/config");
+const jwt = require("jsonwebtoken");
+
+module.exports = function () {
+  passport.use(
+    new GoogleTokenStrategy(
+      {
+        clientID: config.googleAuth.clientID,
+        clientSecret: config.googleAuth.clientSecret,
+      },
+      async function (accessToken, refreshToken, profile, done) {
+        let email = profile.emails[0].value;
+        let user_name = profile.name.givenName;
+        let display_name = profile.displayName;
+        let result = await pool.query({
+          text: " select roles.role,users.email,organizations.name from users,roles,organizations where users.email = '" + email + "'"/*eslint-disable-line prettier/prettier */,
+        });
+        if (result.rowCount) {
+          let token_email = result.rows[0].email;
+          let role = result.rows[0].role;
+          let organization_data = result.rows[0].name;
+          var exp_time = {
+            "expiresIn"/*eslint-disable-line prettier/prettier */: process.env.JWT_expire_time, //eslint-disable-line  no-undef
```

Comment:
why you disabled prettier/prettier

Reply:
it's because we had followed standard to create json object now changed that